addons/website_sale/models/product_template.py

```diff
@@ -272,13 +272,6 @@ def _get_combination_info(self, combination=False, product_id=False, add_qty=1,
                 combination_info['price_extra'], product_taxes, taxes, company_id, pricelist,
                 product, partner
             )
-            base_unit_price = product._get_base_unit_price(list_price)
-            if pricelist.currency_id != product.currency_id:
-                base_unit_price = pricelist.currency_id._convert(
-                    base_unit_price,
-                    pricelist.currency_id,
-                    company_id,
-                    fields.Date.today())
```

Comment:
I see why you took that off, as this is dead code. Nonetheless, it has nothing to do here, especially not in the same commit. This clean dead code unrelated to the fix, in a file not even touched by the fix... :thinking:

Reply:
Agreed, moved to another commit. Thanks for the review.